server/db/models/game.js

```diff
@@ -333,6 +328,45 @@ gameSchema.methods.initializeGame = function (){
   this.dealCards();
 };
 
+gameSchema.methods.pushGameState = function(){
+  var publicPlayerArray = this.players.map(function(player){
+    var p;
+    p._id = player[i]._id;
```

Comment:
what is the [i] supposed to capture? should it just be player._id since the map is looping through each player in the game?